django-verdant/rcasite/settings/production.py

```diff
@@ -102,6 +96,52 @@
 if 'MEDIA_DIR' in env:
     MEDIA_ROOT = env['MEDIA_DIR']
 
+
+# Email settings
+# We use SMTP to send emails. We typically use transactional email services
+# that let us use SMTP.
+# https://docs.djangoproject.com/en/2.1/topics/email/
```

Comment:
**nitpick**
Current Django==1.9.13, so the comment link is incorrect

Reply:
ah, well spotted